src/test/java/io/netty/incubator/codec/http3/AbtractHttp3ConnectionHandlerTest.java

```diff
@@ -40,11 +40,16 @@ public void testOpenLocalControlStream() throws Exception {
         handler.channelRegistered(ctx);
         handler.channelActive(ctx);
 
+        final EmbeddedQuicStreamChannel localControlStream = quicChannel.localControlStream();
+        assertNotNull(localControlStream);
+
         assertNotNull(Http3.getLocalControlStream(quicChannel));
 
         handler.channelInactive(ctx);
         handler.channelUnregistered(ctx);
         handler.handlerRemoved(ctx);
+
+        localControlStream.finishAndReleaseAll();
```

Comment:
you may also want to assert the return value.. True for all calls :)

Reply:
Fixed in #141